docs/getting_started/build_an_app.md

```diff
@@ -0,0 +1,378 @@
+# Build an App
+This guide will guide you through the steps of how to build a Lumen app that will look like this:
+
+![](../_static/getting_started/build_app_07.png)
+
+
+
+
+## Building the app
+This section will walk through the steps to creating the app.
+Create a file named `penguins.yaml`.
+The first thing which is needed is a source. Here we will add a penguin source from a file that contains a penguin's table.
+
+Then launch the app with `lumen serve penguins.yaml --autoreload` in your terminal and open `http://localhost:5006` in your browser of choice.
+By using `--autoreload`, the dashboard automatically refreshes and updates the application whenever we make changes to the YAML file.
+
+
+::::{tab-set}
+:::{tab-item} YAML
+:sync: yaml
+``` {code-block} yaml
+
+sources:
+  penguin_source:
+    type: file
+    tables:
+      table_penguin: https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2020/2020-07-28/penguins.csv
+```
+:::
+
+:::{tab-item} Preview
+:sync: preview
+
+![](../_static/getting_started/build_app_00.png)
+:::
+::::
+
+This returns an empty dashboard because we don't have a targeted view.
+The simplest view to add is a table with the penguin's table in it.
```

Comment:
Can you keep paragraph together rather than starting each sentence on a new line.

Reply:
I like having sentences separated by a new line; the output will be the same when converted to HTML. 

I think it has some advantages (even though they are small):
- For version control, it is easier to see what sentences have changed.  
- When staging changes on a large paragraph with multiple changes across sentences, I have a tendency to either accept or discard all the changes, new lines give me more fine-grained control. 
- When using spell checking, I can easily see what sentence has a spelling error based on the line.

Reply:
Those are all good points @Hoxbro though I see a few downsides too:

1. This style hurts the fluidity of readability when looking at the Markdown file.
2. Is this something we would try to enforce throughout or is this something you only want to do for yourself when working on the docs?

As I think this would be hard to enforce throughout, it would be fine with me if @Hoxbro uses this approach for himself as long as there is no expectation that other people will follow this style (and people could rewrap the text if they want). The downside with *that* is inconsistency...

My only other suggestion might be that this style of separating sentences might be a good idea while working on a docs PR but we should rewrap the text in a consistent way before merge. How does that sound?

Reply:
Right readability and consistency are my main concerns too. I agree with all your points @Hoxbro but if we decide to do this then I think we should do it consistently and it just hurts my eyes a little too much to agree to that.

Reply:
Let's get @droumis' take here as well.

Reply:
So for consistency, should we agree on a rewrap width before merging docs PRs? Before the merge, I'm ok with any style...

Reply:
I don't think it needs to be as black and white as both of you make it out to be. I think having different "formats" across the markdown files is fine, as long as the final product (the HTML) is consistent.  

For readability, I don't think it is as big of an eyesore, it is different but you get used to it quickly.

Reply:
Leaving it for now but still leaning towards wrapping the paragraphs.